IM_calculation/IM/intensity_measures.py

```diff
@@ -58,12 +58,11 @@ def get_SDI_nd(acceleration, period, NT, DT, z, alpha, dy, dt):
     if acceleration.ndim != 1:
         ts, dims = acceleration.shape
         Nstep = calculate_Nstep(DT, NT)
-        displacements = np.zeros((period.size, Nstep - 1, dims))
+        displacements = np.zeros((period.size, Nstep, dims))
```

Comment:
what was the reason for changing the shape? 
i have a vague memory that it was set to Nt -1 for a reason

Reply:
Thought Nstep -1 should be correct as it is "displacements". It turned out sometimes it can be Nstep, and other times Nstep - 1. 
It was due to rounding. I added comment in Burks_Baker_2013_elastic_inelastic.py

Reply:
have you tested this on ASCII data?
because last time similar thing happened and it was because of the float32 and float64 precision difference.
the solution was to kept the old shape but force the dt into float32
see :
https://github.com/ucgmsim/IM_calculation/blob/ffd4997c399296b91c8ef1afe0b5cb8d2107eb20/IM_calculation/IM/read_waveform.py

Reply:
Testing with ASCII is a good call, Jesse encountered some bugs due to Nstep - 1 and that's the reason why it is what it is in pSA at the moment.

Reply:
Yes, tested with both binary and ASCII.